pages/index.js

```diff
@@ -74,93 +81,52 @@ export default function Home() {
     [email]
   );
 
-  const getRandomImages = (count, exhibits) =>
-    Array(count)
-      .fill()
-      .map(() => {
-        const exhibitNames = Object.keys(exhibits);
-        const randomExhibit =
-          exhibitNames[Math.floor(Math.random() * exhibitNames.length)];
-        const randomExhibitImages = exhibits[randomExhibit];
-        return randomExhibitImages[
-          Math.floor(Math.random() * randomExhibitImages.length)
-        ];
-      });
-
-  const refreshUser = useCallback(async () => {
-    console.info("Refreshing the user");
-    const refreshExhibits = async (currentUser, exhibitNames) => {
-      const exhibitsResponses = await Promise.all(
-        exhibitNames.map((exhibit) =>
-          vanaGet(`account/exhibits/${exhibit}`, {}, authToken).then(
-            (response) => ({ name: exhibit, response })
-          )
-        )
-      );
+  // Get random user exhibit images
+  const populateRandomUserExhibits = useCallback(async (token) => {
+    const images = await getRandomUserExhibits(token, 3);
 
-      const newUser = exhibitsResponses.reduce(
-        (user, { name, response }) => {
-          if (response.success) {
-            user.exhibits[name] = response.urls;
-          }
-          return user;
-        },
-        { ...currentUser, exhibits: currentUser.exhibits ?? {} }
-      );
+    setRandomExhibitImages(images);
+  }, []);
 
-      setUser(newUser);
+  // Get Text to Image exhibit images
+  const populateTextToImageExhibits = useCallback(async (token) => {
+    async function refreshImages() {
+      const images = await getTextToImageUserExhibits(token);
 
-      Object.keys(newUser.exhibits).length &&
-        setRandomExhibitImages(getRandomImages(3, newUser.exhibits));
-    };
+      setTextToImageExhibitImages(images);
```

Comment:
We need to reverse the images to show the last generated images on the top.

```suggestion
      setTextToImageExhibitImages(images.reverse());
```